src/components/ToolBar.tsx

```diff
@@ -1,24 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../index.scss';
 import gear from '../assets/icon-gear.svg';
 import plus from '../assets/shared/icon-plus.svg';
 
-export const ToolBar = () => {
+interface IProps {
+  callback(id: string): void;
+  sortOption: string;
+  check: any;
+}
+
+export const ToolBar = (props: IProps) => {
+  const { callback, sortOption } = props;
+  const [hiddenState, setHiddenState] = useState<string>('hidden');
+
+  const toggleSort = (): void => {
+    if (hiddenState === 'hidden') {
+      setHiddenState('');
+    } else {
+      setHiddenState('hidden');
+    }
+  };
+
   return (
     <div className="toolbar">
       <aside className="toolbar-left">
         <div>
           <img src={gear} alt="gear icon" />
           <span>6 Suggestions</span>
         </div>
-        <div className="toolbar-left-sort">
-          <span>Sort By:</span>
-          <select name="sortBy">
-            <option value="1">Most Up Votes</option>
-            <option value="1">1</option>
-            <option value="1">1</option>
-            <option value="1">1</option>
-          </select>
+        <div className={`toolbar-left__sort`} onClick={toggleSort} id="sort">
+          <span>Sort By: {sortOption}</span>
+          <div className={`${hiddenState}`}>
+            <button
+              onClick={() => callback('most-upvotes')}
+              id="most-upvotes"
+              className={sortOption === 'Most Up Votes' ? 'check' : ''}
```

Comment:
Usually, it's a good idea to add `type="button"` on these buttons because the default behavior of a button is "submit".